features/pdf-viewer.yml

```
@@ -0,0 +1,9 @@
+name: pdfViewerEnabled
+description: "The `navigator.pdfViewerEnabled` read-only property indicates whether the browser supports inline display of PDF files when navigating to them. If inline viewing is not supported the PDF is downloaded and may then be handled by an external application."
```

The external application stuff is not in scope for us. Edits for brevity

```suggestion
description: The `navigator.pdfViewerEnabled` read-only property returns a Boolean for whether the browser navigates to and shows a PDF in the browser window or downloads the PDF.
```